lib/polyfill/mediasource.js

```diff
@@ -35,19 +35,12 @@ shaka.polyfill.MediaSource = class {
     } else if (safariVersion && window.MediaSource) {
       // NOTE:  shaka.Player.isBrowserSupported() has its own restrictions on
       // Safari version.
-      if (safariVersion <= 10) {
+      if (safariVersion <= 12) {
+        shaka.log.info('Patching Safari 11 & 12 MSE bugs.');
```

Comment:
Where it mentions Safari 11 & 12 it should be Safari 8, 9, 10, 11 & 12 or older versions of Safari, or something like that. 

(there are several lines in this file but I only put this comment)